### PR TITLE
[ui] Add minimalist gallery, clicking still opens jolla-gallery

### PR DESCRIPTION
**Pull request by vknecht**

### CODE REVIEW

File: qml/pages/GalleryUI.qml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Media 1.0
+import QtMultimedia 5.4
+import QtQuick.Layouts 1.0
+import uk.co.piggz.harbour_advanced_camera 1.0
+import Nemo.Configuration 1.0
+import "../components/"
```
Comment (mlehtima) [inline qml/pages/GalleryUI.qml]: Many unused imports, please leave only those that are used in this file

File: qml/pages/GalleryUI.qml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Page {
+    id: galleryPage
+
+    property var photoList: ({})
+
+    // The effective value will be restricted by ApplicationWindow.allowedOrientations
+    allowedOrientations: Orientation.Landscape
```
Comment (mlehtima) [inline qml/pages/GalleryUI.qml]: Would be nice to support all orientations.

File: qml/pages/CameraUI.qml [hunk 3]
```diff
@@ -608,4 +629,26 @@ Page {
...
     }
+
+    RoundButton {
+        id: btnGallery
+
+        visible: false
+        enabled: false
+
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: 50
```
Comment (mlehtima) [inline qml/pages/CameraUI.qml]: Hardcoding margins with magic number is not a proper way, we should use Silica margin values when possible.

File: qml/pages/CameraUI.qml [hunk 4]
```diff
@@ -608,4 +629,26 @@ Page {
 
         return siz.width + "x" + siz.height;
     }
+
+    RoundButton {
+        id: btnGallery
+
+        visible: false
+        enabled: false
```
Comment (mlehtima) [inline qml/pages/CameraUI.qml]: These seem to be always set to the same value so maybe using "enabled: visible" and in other parts of the code only setting the visible value would make code a little simpler.

File: qml/pages/GalleryUI.qml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+                        console.log("Clicked", thumbnail.source)
+                        Qt.openUrlExternally(thumbnail.source)
+                    }
+                }
+            }
+
+        }
+    }
+
+
```
Comment (mlehtima) [inline qml/pages/GalleryUI.qml]: Cleanup useless empty rows. Between items these can be used for readability but no need to have these here and some other places.

File: qml/pages/GalleryUI.qml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    allowedOrientations: Orientation.Landscape
+
+    SlideshowView {
+        id: gallery
+
+        clip: true
+        width: parent.width
+        height: parent.height
+        itemWidth: width / 2
+        itemHeight: height * 2 / 3
```
Comment (mlehtima) [inline qml/pages/GalleryUI.qml]: Why not have images show in fullscreen size?

File: qml/pages/GalleryUI.qml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+        y: parent.height / 6
+
+        model: photoList
+
+        delegate: Rectangle {
```
Comment (mlehtima) [inline qml/pages/GalleryUI.qml]: Do we need borders for the images?

File: qml/pages/CameraUI.qml (hunk 3 above)
Comment (vknecht) [reply qml/pages/CameraUI.qml]: Not sure how to handle this one...

**Pull request merged**